src/Services/UserService.ts

```diff
@@ -1,29 +1,44 @@
 import * as bcrypt from "bcryptjs";
-import logger from "./Logger";
+import * as jwt from "jsonwebtoken";
+import config from "../Config/Config";
 
 import {User} from "../Models/User";
+import {ServiceError} from "./ServiceError";
 
-export class UserService {
+export class UserService extends ServiceError {
     // eslint-disable-next-line
-    public async registerUser(email: string, hash_password: string) {
+    public async registerUser(email: string, password: string) {
+        const user = await User.findOne({
+            raw: true,
+            where: {
+                email,
+            }
+        });
+
+        if (user) {
+            throw new ServiceError(409, "There is already a user with this email address.");
+        }
+        // eslint-disable-next-line
+        const hash_password = bcrypt.hashSync(password, 10);
         // eslint-disable-next-line
         return await User.create({email, hash_password});
```

Comment:
I think that instead of disabling eslint, you can do this:
```
const hashPassword = bcrypt.hashSync(password);
return Await User.create({email, 'hash_password': hashPassword});
```